style.css

```diff
@@ -0,0 +1,33 @@
+.morning {
+    text-align: center;
+}
+
+img {
+    max-width: 60%;
```

Comment:
If you want all images to generally be 60% width, you can just say that and avoid `max-width`:
```suggestion
    width: 60%;
```